Week_01_Basics/Assignment/resume.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<link href="./stylesheet.css" rel="stylesheet">
+
+<title>Chevaun Ho's Resume and Experience</title>
+
+<head>
+	<header>
+		<h1>Chevaun&middotHo</h1>
+		<h4>SYDNEY-BASED CREATIVE</h4>
+		<h3>painter &middot journalist &middot social media producer &middot front end web developer</h3>
+
+		<!-- Site navigation menu -->
+		<nav>
+			<div class="navbar">
+				<ul>
+					<li><a href="portfolio.html">about me</a></li>
+					<li><a href="resume.html">resume</a></li>
+					<li><a href="contactme.html">contact me</a></li>
+			</div class="nav-wrapper">
+		</nav>
+	</header>	
+</head>
+<br/>
+<hr>
+<br/>
+<body>
+	<table>
+	  <tr>
+	  	<td>
+			<h2>Educational achievements...</h2>
+				<p>&middot Front End Web Development, General Assembly </p>
```

Comment:
You can use a list here to achieve the middot.  Otherwise the middot is correctly used as ```&middot;```